web/src/test/auto/headless/engine/js-processor/bundled-module.tests.js

```diff
@@ -1,7 +1,7 @@
 import { assert } from "chai";
-import * as Package from "keyman/engine/js-processor";
-import * as Package2 from "keyman/engine/keyboard";
-const KMWString = Package2.KMWString;
+import * as JSProcessorPackage from "keyman/engine/js-processor";
+import * as KeyboardPackage from "keyman/engine/keyboard";
+const KMWString = KeyboardPackage.KMWString;
```

Comment:
Yeah, those are better names, but surely these should be called `JSProcessorModule` and `KeyboardModule`?